src/Esprima/Utils/ToJavascriptConverter.cs

```diff
@@ -0,0 +1,1585 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Esprima.Ast;
+
+namespace Esprima.Utils
+{
+    public static class ToJavascriptConverterExtension
+    {
+        public static string ToJavascript(this Node node, bool beautify = false)
+        {
+            return ToJavascriptConverter.ToJavascript(node, beautify);
+        }
+    }
+
+    public class ToJavascriptConverter

```

Comment:
is this implementation details, like `internal sealed`, would name like `JavascriptWriter` or something like so make more sense?